.changeset/dirty-mirrors-retire.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-compat-api': patch
+'example-app-next': patch
+'example-app': patch
```

Comment:
`example-app`* packages can be excluded as they are really just for debugging purposes in the main repository & not versioned/packaged - could have highlighted that in the review earlier, sorry.

```suggestion
```

Reply:
ok - just going to quickly apply the change to unblock this.